packages/x-rollup/src/rollup-config.js

```diff
@@ -44,11 +44,17 @@ module.exports = ({ input, pkg }) => {
 				input,
 				external,
 				plugins: [
+<<<<<<< HEAD
```

Comment:
@andygout unresolved merge conflict

Reply:
Good catch. I will just do an amend commit on top of the existing one.